suppressible-error-prone/src/main/java/com/palantir/suppressibleerrorprone/VisitorStateModifications.java

```diff
@@ -17,25 +17,37 @@
 package com.palantir.suppressibleerrorprone;
 
 // CHECKSTYLE:OFF
+
 import com.google.errorprone.VisitorState;
-import com.google.errorprone.fixes.SuggestedFix;
 import com.google.errorprone.matchers.Description;
+import com.sun.source.tree.AnnotationTree;
 import com.sun.source.tree.ClassTree;
+import com.sun.source.tree.IdentifierTree;
+import com.sun.source.tree.MemberSelectTree;
+import com.sun.source.tree.MethodTree;
+import com.sun.source.tree.ModifiersTree;
 import com.sun.source.tree.Tree;
+import com.sun.source.tree.VariableTree;
 import com.sun.source.util.TreePath;
-import com.sun.tools.javac.util.JCDiagnostic.DiagnosticPosition;
+import java.util.Map;
 import java.util.Optional;
+import java.util.WeakHashMap;
 import java.util.logging.Logger;
 import java.util.stream.Stream;
+import javax.lang.model.element.Name;
 // CHECKSTYLE:ON
 
 public final class VisitorStateModifications {
     private static final Logger log = Logger.getLogger(VisitorStateModifications.class.getName());
 
+    // Weak map so that we don't leak memory by keeping our mutable fixes around forever, once error-prone has done
+    // with them, they can be safely collected.
```

Comment:
```suggestion
    // Weak map so that we don't leak memory by keeping our mutable fixes around forever, once errorprone
    // finished with a class, our SuppressingFix's can be safely collected.
```